src/loadMessages.js

```diff
@@ -0,0 +1,11 @@
+export default function loadMessages(lang) {
+  const wrapped = import(`./messages/${lang}`)
+
+  if (typeof window !== 'undefined') {
+    return wrapped.then(msgs => msgs.default)
+  }
+
+  /* eslint-disable global-require, import/no-dynamic-require, prefer-template */
+  // return Promise.resolve(require('./messages/' + lang).default)
+  return __webpack_require__(wrapped.resolve()).default
+}
```

Comment:
Yes. As I say there is no way to disable the plugin only on sections of the code. It is applied to all code.